File: app/src/main/java/com/hoymm/root/tictactoe2/DisplayMenu/MainMenu/MainMenuDisplay.java
package com.hoymm.root.tictactoe2.DisplayMenu.MainMenu;

import android.support.v7.app.AppCompatActivity;

import com.hoymm.root.tictactoe2.DisplayMenu.DisplayMenu;

/**
 * Created by Damian Muca - Kaizen (15.09.17)
 */

public class MainMenuDisplay extends DisplayMenu {

    @Override
    protected void initFragmentObjects(AppCompatActivity activity) {
        setHeaderFrg(new MainMenuTextAndGraphHeaderFragment());
        setButtonsFrg(new MainMenuButtonsFragment());
    }
}

File: app/src/main/java/com/hoymm/root/tictactoe2/DisplayMenu/MainMenu/MainMenuDisplay.kt
package com.hoymm.root.tictactoe2.DisplayMenu.MainMenu

import android.support.v7.app.AppCompatActivity

import com.hoymm.root.tictactoe2.DisplayMenu.DisplayMenu

/**
 * Created by Damian Muca - Kaizen (15.09.17)
 */

class MainMenuDisplay : DisplayMenu() {

    override fun initFragmentObjects(activity: AppCompatActivity) {
        headerFrg = MainMenuTextAndGraphHeaderFragment()
        buttonsFrg = MainMenuButtonsFragment()
    }
}
